contents/handbook/people/team-structure/marketing/objectives.mdx

```diff
@@ -1,12 +1,11 @@
 * **Objective**:
-    * Accelerate word of mouth growth by significantly increasing awareness of what PostHog does as a company and as a product
+    * Do it again, but with new channels. 
```

Comment:
Do we want to repeat "Accelerate word of mouth growth by significantly increasing awareness of what PostHog does as a company and as a product" because it could be unclear what "it" is?

Reply:
Yeah agreed, this isn't clear enough. I've tried an updated objective - what do you think?

Reply:
Yup, looks good I think